sdk/storage/Azure.Storage.Blobs.ChangeFeed/tests/ChangeFeedTestBase.cs

```diff
@@ -12,13 +12,24 @@
 
 namespace Azure.Storage.Blobs.ChangeFeed.Tests
 {
+    [ClientTestFixture(
+    BlobClientOptions.ServiceVersion.V2019_02_02,
+    BlobClientOptions.ServiceVersion.V2019_07_07,
+    BlobClientOptions.ServiceVersion.V2019_12_12,
+    BlobClientOptions.ServiceVersion.V2020_02_10,
+    BlobClientOptions.ServiceVersion.V2020_04_08,
+    BlobClientOptions.ServiceVersion.V2020_06_12,
+    BlobClientOptions.ServiceVersion.V2020_08_04,
```

Comment:
Do we need all of these ? CF hasn't GA, I think it's fine to just stick to latest here.

Reply:
We do not, will fix.

Reply:
We also don't need all the service versions in the Common Test Base.

Reply:
Fixed.